src/SmartComponents/Inventory/applications/vulnerabilities/DataMapper.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { SecurityIcon } from '@patternfly/react-icons';
+
+const colorToImpact = {
+    High: '#ec7a08',
+    Important: '#ec7a08',
+    Medium: 'var(--pf-global--warning-color--100)',
+    Moderate: 'var(--pf-global--warning-color--100)',
+    Critical: 'var(--pf-global--danger-color--100)',
+    Low: 'var(--pf-global--BackgroundColor--disabled)'
+};
+
+export function createCveListBySystem({ isLoading, ...rest }) {
+    if (!isLoading) {
+        const { payload: { data, meta }} = rest;
+        return {
+            data: data.map(row => ({
+                id: row.id,
+                cells: [
+                    <span key={ row.id }>
+                        <SecurityIcon size="md" color={ colorToImpact[row.attributes.impact] } />
+                    </span>,
+                    row.attributes.synopsis,
+                    `${row.attributes.description.substr(0, 199)}...`,
```

Comment:
Wondering if limiting the description size  to 200 characters isn't better done by setting `text-overflow` 🤔 also, is the full text available somewhere else?  if not, maybe this should also have a popup (containing full text)

Reply:
I'm gonna wrap that string in span and add the whole unchanges string as title.